netsocadmin/routes/tools.py

```diff
@@ -183,13 +188,16 @@ def dispatch_request(self) -> str:
         # Check that all fields are valid
         valid, msg = self.validate(username, password, dbname)
         if not valid:
+            self.logger.debug(f"error {msg}")
```

Comment:
Should this also be self.logger.error(...) ?

Reply:
Yea, any error should do that. Also try to add meaningful error messages :)